app/models/work_version.rb

```diff
@@ -31,7 +31,18 @@ class WorkVersion < ApplicationRecord
                  identifier: [:string, array: true, default: []],
                  based_near: [:string, array: true, default: []],
                  related_url: [:string, array: true, default: []],
-                 source: [:string, array: true, default: []]
+                 source: [:string, array: true, default: []],
+                 owner: :string,
+                 manufacturer: :string,
+                 model: :string,
+                 instrument_type: :string,
+                 measured_variable: :string,
+                 available_date: :string,
+                 decommission_date: :string,
```

Comment:
In the EdtfDate module `#valid?` required the date to be a valid and present to return true so an empty available and/or decommission date would throw an error. This feels redundant because `presence: true` on `published_date` already handles this. I modified it so the module just validates format, allowing for blank values, and let `presence: true` handle that validation as needed.